seed/seed.json

```diff
@@ -57,6 +57,34 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
         },
+        {
+            "name": "DeAMPStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 0,
+                    "feature_association": {
+                        "enable_feature": ["BraveDeAMP"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 100,
+                    "feature_association": {
+                        "disable_feature": ["BraveDeAMP"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "100.1.39.5",
```

Comment:
shouldn't it be 101. ?

Reply:
Hmm, I thought I'd taken it from brave://system and 101 sounds right. Should I make the change @iefremov?